tvbwidgets/ui/head_widget.py

```diff
@@ -140,6 +147,11 @@ def __draw_mesh_actor(self, surface, config):
         if config is None:
             config = HeadWidgetConfig(name='Surface-' + str(surface.number_of_vertices))
 
+        for prev_config in self.existent_configs:
+            if config.is_incompatible(prev_config):
+                self.logger.info("HeadWidget can not support multiple surfaces with incompatible coloring!")
+                return
+
```

Comment:
I think we should move the current new line 117 after this `for`.
To add to the list of `existent_configs` only in case the `prev_config ` is compatible, thus displayed.
`self.existent_configs.append(prev_config)`
Otherwise, in the current form, after the first incompatible config, we will not be able to add any new surfaces, even if compatible with the first configs

Reply:
if we append to the existent configs just after the for, at the first call of : self.existent_configs.append(prev_config), prev_config will not be assigned. Will it be ok to append the current config before the For?